README.md

```diff
@@ -32,3 +32,31 @@ found at the following links:
 
 + https://help.github.com/articles/creating-a-pull-request/ 
 + https://help.github.com/articles/autolinked-references-and-urls/
+
+
+## How to set up the API
+1. Before you can access the API, you need to install docker toolbox and docker-for-Mac (or whichever is appropriate for your machine)
+
+    https://www.docker.com/get-docker
+
+2. Once you have those installed, run the following command in your terminal:
+```sh 
+touch .env
+```
+This gives you access to the API's .env file. 
+
+3. Next you'll need to run the following:
+```sh
+  docker-compose build
+  docker-compose run --rm web bin/setup
+  docker-compose up web 
+```
+This last command starts the server. You'll need to keep this running in a separate tab.
+
+4. Finally, once that is up, run the following command in a separate tab to get a JSON web token to use in Postman.
+```sh 
+docker-compose run --rm web rake token
+```
+In Postman, you will need to add a header with `Authorization` as the key and `Bearer <your-token-here>` as the value.
+
+You should also be able to go to `localhost:3000` on your computer. 
```

Comment:
Since we're suggesting Postman, maybe this could be something along the lines of:

"You should now be able to send requests to `localhost:3000`."